.github/util/initialize/action.yml

```diff
@@ -17,20 +17,55 @@ runs:
       with:
         node-version: "${{ inputs.node-version }}"
 
+    # See: https://github.com/dart-lang/sdk/issues/52266
+    - run: Invoke-WebRequest https://pub.dev
+      if: runner.os == 'Windows'
+      shell: powershell
+
+    # See: https://github.com/orgs/community/discussions/131594
+    # The composite action requires an explict shell, but bash is not available on windows-arm64 runner.
+    # For the following commands conditionally use bash or powershell based on the runner.os:
     - run: dart pub get
+      if: runner.os != 'Windows'
       shell: bash
 
+    - run: dart pub get
+      if: runner.os == 'Windows'
+      shell: powershell
```

Comment:
as the command is the same in both case, this could be simplified by letting the default shell be used.

Reply:
That’s already refactored in a later PR.